CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.0.0] - 2024-02-xx
```

Comment:
this is not a new major version imho. when someone who is on PHP 8.0 installs the package it will use the previous version automatically; so it is not breaking. it "just" adds the flexibility to also use psr/cache >= 1.0